### PR TITLE
docstring

### PR DESCRIPTION
**Pull request by YufengXin**
#70.

@sajith already made the constant flag name change.
Used docstring in a few places

### CODE REVIEW

**Comment by YufengXin** [comment]
black reformatting is different in 3.8 (my Mac) and 3.9 (here)?

**Comment by YufengXin** [comment]
updated my MacOS to Python3.11

File: src/sdx/pce/utils/random_topology_generator.py [hunk 1]
```diff
@@ -104,7 +106,8 @@ def link_property_assign(self):  # Pass in the bw name
 
         return self.latency_list
 
-    # if u and v connected
+    """ if u and v connected """
+
     def nodes_connected(self, g, u, v):
```
Comment (sajith) [inline src/sdx/pce/utils/random_topology_generator.py:111]: This too:

```suggestion
    def nodes_connected(self, g, u, v):
        """ if u and v connected """
```

File: src/sdx/pce/utils/functions.py [hunk 2]
```diff
@@ -79,7 +79,9 @@ def biconnectivity(self):
         pass
 
 
-# use dijsktra to get the primary shortest path
+""" use dijsktra to get the primary shortest path """
+
+
 def dijnew(graph, start_node, end_node):
```
Comment (sajith) [inline src/sdx/pce/utils/functions.py:85]: Docstrings should be below the method name:

```suggestion
def dijnew(graph, start_node, end_node):
    """ use dijsktra to get the primary shortest path """
```

File: src/sdx/pce/utils/functions.py [hunk 3]
```diff
@@ -122,7 +124,9 @@ def dijnew(graph, start_node, end_node):
     return path
 
 
-# make the non-simple graph to be the simple graph
+""" make the non-simple graph to be the simple graph """
+
+
 def graph_simplify(graph):
```
Comment (sajith) [inline src/sdx/pce/utils/functions.py:130]: Another suggestion:

```suggestion
def graph_simplify(graph):
    """ make the non-simple graph to be the simple graph """
```

File: src/sdx/pce/utils/functions.py [hunk 4]
```diff
@@ -135,7 +139,9 @@ def graph_simplify(graph):
     return graph
 
 
-# remove the primary shortest path and redo the dijsktra to get the backup path
+""" remove the primary shortest path and redo the dijsktra to get the backup path """
+
+
 def backup_path(graph, start_node, end_node):
```
Comment (sajith) [inline src/sdx/pce/utils/functions.py:145]:
```suggestion
def backup_path(graph, start_node, end_node):
    """ remove the primary shortest path and redo the dijsktra to get the backup path """
```

File: src/sdx/pce/utils/random_topology_generator.py [hunk 5]
```diff
@@ -92,8 +93,9 @@ def generate_graph(self, plot=True, g=None) -> nx.Graph:
 
         return self.graph
 
-    # set the random bw and latency per link
-    def link_property_assign(self):  # Pass in the bw name
+    """ set the random bw and latency per link """
+
+    def link_property_assign(self):
```
Comment (sajith) [inline src/sdx/pce/utils/random_topology_generator.py:98]:
```suggestion
    def link_property_assign(self):
        """ set the random bw and latency per link """
```

**Comment by YufengXin** [comment]
Thanks. Fixed. I've updated all the tools after Python upgrade yesterday

**Pull request merged**